app/src/main/res/values/strings.xml

```diff
@@ -41,27 +41,27 @@
     <string name="records">records</string>
     <string name="no_data">No data found</string>
     <string name="enter_money_donated">Enter Money Donated</string>
-    <string name="people_who_registered_for_pooja">People Who Registered For Pooja</string>
-    <string name="people_who_donated_money_for_temple">People Who Donated Money For Temple</string>
+    <string name="people_who_registered_for_pooja">People who registered for pooja</string>
+    <string name="people_who_donated_money_for_temple">People Who donated money for temple</string>
     <string name="employee_id">Employee ID</string>
     <string name="delete">Delete</string>
-    <string name="register_for_new_pooja">Register For New Pooja</string>
-    <string name="donate_money_to_temple">Donate Money To Temple</string>
+    <string name="register_for_new_pooja">Register for new pooja</string>
+    <string name="donate_money_to_temple">Donate money to temple</string>
     <string name="insert_">Insert</string>
-    <string name="list_of_donated_people_who_paid_only">List Of Donated People: who paid only</string>
-    <string name="list_of_donated_people_who_not_paid_only">List Of Donated People: who not paid only</string>
-    <string name="list_of_registered_poojas_people_who_paid_only">List Of Registered Poojas: people who paid only</string>
-    <string name="list_of_registered_poojas_who_not_paid_only">List Of Registered Poojas: who not paid only</string>
+    <string name="list_of_donated_people_who_paid_only">People who are willing to donate and have paid</string>
+    <string name="list_of_donated_people_who_not_paid_only">People who are willing to donate and have not paid</string>
+    <string name="list_of_registered_poojas_people_who_paid_only">People who have registered and have paid</string>
+    <string name="list_of_registered_poojas_who_not_paid_only">People who have registered and have not paid</string>
     <string name="find">Find</string>
-    <string name="update_registered_pooja_details">Update Registered Pooja Details</string>
-    <string name="update_the_details_of_donated_money">Update The Details Of Donated Money</string>
+    <string name="update_registered_pooja_details">Update registered pooja details</string>
+    <string name="update_the_details_of_donated_money">Update the details of donated money</string>
     <string name="update">Update</string>
     <string name="error_notSelected">Please select any option</string>
     <string name="fetch_name">name</string>
     <string name="insert_data">Insert Data</string>
     <string name="nothingselected">Nothing selected</string>
     <string name="custompooja">Custom Pooja</string>
-    <string name="amount_in_rs">Amount:                                         Rs.</string>
+    <string name="amount_in_rs">Amount:Rs.</string>
```

Comment:
revert this change, it has a particular reason for giving some spaces
find it in `ReadSingleData` activity